src/main.py

```diff
@@ -112,19 +115,35 @@ async def on_message(message: Message) -> None:
 
         await bc.play(message, skip_count)
 
-    elif message_text.startswith("!form"):
-        if not bc:
-            await message.channel.send("You need to use !list first.")
-            return
-
-        # Pull the Google Drive link to the form image from the message (if it exists)
-        command_args = message.content.split()[1:]
-        if command_args:
-            await voting.generate_form(
-                bc, message, google_drive_image_link=command_args[0]
-            )
+    elif message_text.startswith("!image"):
+        message_split = message.content.split()
+        if len(message_split) > 1:
+            arg1 = message_split[1]
+            # See if it's a clear command or giving a URL
+            if arg1 == "clear":
+                loaded_image = None
+            else:
+                loaded_image = arg1
+            await message.add_reaction(config.COMMAND_SUCCESS)
+        elif len(message.attachments) > 0:
+            # TODO: Some basic validity filtering
+            loaded_image = message.attachments[0].url
+            await message.add_reaction(config.COMMAND_SUCCESS)
         else:
-            await voting.generate_form(bc, message)
+            if loaded_image is not None:
+                message_reply_content = (
+                    f"Loaded image: {loaded_image}\n\nTo change this image, "
+                    "either run `!image <image_url>` or"
+                    " just `!image` with a valid media attachment. "
+                    "To clear this image, run `!image clear`"
+                )
+            else:
+                message_reply_content = (
+                    "No image is currently loaded.\n\nTo add an image, "
+                    "either run `!image <image_url>` or"
+                    " just `!image` with a valid media attachment.\n"
```

Comment:
Best to be consistent on where the space is put for multi-line strings. Either at the beginning or the end of each string. Here they're mixed.